boards/nucleo144-f413/include/periph_conf.h

```diff
@@ -130,10 +157,8 @@ static const uart_conf_t uart_config[] = {
         .tx_af      = GPIO_AF7,
         .bus        = APB1,
         .irqn       = USART2_IRQn,
-#ifdef UART_USE_DMA
-        .dma_stream = 4,
-        .dma_chan   = 4
-#endif
+        .dma        = 3,
+        .dma_chan   = 4,
```

Comment:
stupid question: how do you select the value used for the channel ? is it global to all streams ?

Reply:
Possible stream/channel for each peripheral are described in the reference manual.